.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
why did you add this file?

Reply:
I think this got added automatically when trying to run the code via vs code, I can remove it